scripts/zones/AlTaieu/Zone.lua

```diff
@@ -19,9 +19,9 @@ require("scripts/globals/missions");
 function onInitialize(zone)
 end;
 
------------------------------------		
--- onConquestUpdate		
------------------------------------		
+----------------------------------- 
+-- onConquestUpdate        

```

Comment:
Trailing whitespace makes Teo sad.
![Image of a Sad Catt](http://cdn.meme.am/instances/500x/61640309.jpg)